src/sentry/sentry_apps/logic.py

```diff
@@ -206,6 +210,48 @@ def _update_events(self) -> None:
             self.sentry_app.events = expand_events(self.events)
 
     def _update_service_hooks(self) -> None:
+        organization_context = organization_service.get_organization_by_id(
+            id=self.sentry_app.owner_id
+        )
+        assert organization_context is not None, "Organization cannot be None for ff"
```

Comment:
The organization here will be the owner of the sentryapp, not the installation orgs. Is that what you want?

Reply:
ye, we want the feature flag check to be based off the owner of the app since the updating updates all installations (i.e is app scoped).